frontend/scripts/react-components/dashboard-element/dashboard-element.selectors.js

```diff
@@ -103,6 +103,15 @@ export const getDynamicSentence = createSelector(
         value: sourcesValue,
         transform: 'capitalize'
       },
+      {
+        panel: 'companies',
+        nodeTypeName: 'ports',
+        id: 'port',
+        prefix: getActivePanelItem('companies', 'PORT') ? 'exported from' : '',
```

Comment:
Please bear in mind that the node types enabled for each of the panels are context-dependent and their names might change / new node types might be added. This won't happen in the nearest data release I believe, so maybe this is not for now, but it would be ideal if we didn't need to hardcode any node type names and instead base this logic on the node type names returned by the filter_meta endpoint.